test/units/testsuite-35.sh

```diff
@@ -258,7 +258,7 @@ cleanup_session() (
 
     systemctl stop getty@tty2.service
 
-    for s in $(loginctl --no-legend list-sessions | awk '$3 == "logind-test-user" { print $1 }'); do
+    for s in $(loginctl --no-legend list-sessions | grep tty | awk '$3 == "logind-test-user" { print $1 }'); do
```

Comment:
Now that almost every user would have an associating manager session, we should add `CLASS` to list-sessions calls to make distinguishing normal sessions easier.

Reply:
yes, I fully agree. i think the choice of fields "list-sessions" outputs right now misses all the interesting fields. To make this efficient we should add a ListSessionsEx() bus call or so that substantially updates the list of fields included. But I think that's stuff for another PR

Reply:
(BTW, are we allowed to change the order/remove fields from list-* output?)

Reply:
that's murky territory. We have changed that in the past. And I think we should do this. Probably not remove, but add should be fine.

We should support --json=/-j mode everywhere though, and for that one stronger requirements apply, because it's pretty much API and not just user visible stuff.